BUILD.md

```diff
@@ -179,14 +179,15 @@ cmake --preset linux -DENABLE_LOGGER=ON
+| Option                    | Description                                                                                                                                                                                                                                                                                                                                                                      | Default                                                                                      |
+|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| `CMAKE_BUILD_TYPE`        | `Debug` builds are generally larger, slower and contain extra correctness checks that will validate game data and interrupt gameplay when problems are detected.<br>`Release` builds are optimized for size and speed and do not include debugging information, which makes it harder to find problems.                                                                          | `Debug`                                                                                      |
+| `BUILD_EDITOR`            | _(Windows-only)_ Build internal editor.                                                                                                                                                                                                                                                                                                                                          | `OFF`                                                                                        |
+| `BUILD_TESTING`           | Enable testing. Requires GTest.                                                                                                                                                                                                                                                                                                                                                  | `OFF`                                                                                        |
+| `DEFAULT_ADDITIONAL_DIRS` | A list of directories that Descent 3 will use as read-only base directories unless `-nodefaultadditionaldirs` is used (see [USAGE.md’s Base directories section](./USAGE.md#base-directories)). This value gets interpreted as a C++ expression, so you’ll need to use C++’s syntax. Here’s an example of a valid value: <pre lang=cpp>`{"C:\\Games\\Descent3\\", "D:\\"}`</pre> | `{}`                                                                                         |
```

Comment:
it would be more idiomatic to provide a list in the [CMake sense](https://cmake.org/cmake/help/latest/command/list.html) and then bracket it and quote it automatically